### PR TITLE
Issue #13421: Add since in javadoc of each property setter for javadoc checks

### PR DESCRIPTION
**Pull request by stoyanK7**
Part of #13421 

---
Adds `@since` to each property setter.

Taken from https://checkstyle.sourceforge.io/checks/javadoc/index.html

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java [hunk 1]
```diff
@@ -196,6 +196,7 @@ public class JavadocVariableCheck
      * Setter to specify the visibility scope where Javadoc comments are checked.
      *
      * @param scope a scope.
+     * @since 6.0
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java:199]: 3.0

But we will catch it in validation

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
@stoyanK7 , please fix CI

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java (hunk 1 above)
Comment (stoyanK7) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java]: Done.

**Comment by stoyanK7** [comment]
@romani Done.

**Pull request merged**